damusTests/LikeTests.swift

```diff
@@ -28,9 +20,30 @@ class LikeTests: XCTestCase {
         let like_ev = make_like_event(keypair: test_keypair_full, liked: liked)!
 
         XCTAssertTrue(like_ev.referenced_pubkeys.contains(test_keypair.pubkey))
-        XCTAssertTrue(like_ev.referenced_pubkeys.contains(cindy))
-        XCTAssertTrue(like_ev.referenced_pubkeys.contains(bob))
+        XCTAssertFalse(like_ev.referenced_pubkeys.contains(cindy))
+        XCTAssertFalse(like_ev.referenced_pubkeys.contains(bob))
+        XCTAssertEqual(like_ev.last_refid()!, id)
+        XCTAssertTrue(like_ev.tags.allSatisfy { !$0[0].matches_char("a") })
+
+        let kindTag = try XCTUnwrap(like_ev.tags.first(where: { $0.count >= 2 && $0[0].matches_char("k") }))
+        XCTAssertTrue(kindTag[1].matches_str("1"))
+    }
+
+    func testReactionLongFormNote() throws {
+        let liked = test_longform_event.event
+        let id = liked.id
+        let like_ev = make_like_event(keypair: test_keypair_full, liked: liked)!
+
+        XCTAssertTrue(like_ev.referenced_pubkeys.contains(test_keypair.pubkey))
         XCTAssertEqual(like_ev.last_refid()!, id)
+
+        let dTagValue = try XCTUnwrap(liked.tags.first { $0[0].matches_char("d") })
+        let aTag = try XCTUnwrap(like_ev.tags.first { $0[0].matches_char("a") })
+        XCTAssertTrue(aTag[1].matches_str("30023:\(test_keypair.pubkey.hex()):\(dTagValue[1])"))
+
+        let kindTag = try XCTUnwrap(like_ev.tags.first(where: { $0.count >= 2 && $0[0].matches_char("k") }))
+//        FIXME(tyiu) the assertion below fails for some reason even though in a different test, I was able to assert kind 1 just fine.
+//        XCTAssertTrue(kindTag[1].matches_str("30023"))
```

Comment:
@jb55 Any idea why this assertion fails? I'm not familiar with the NostrDB stuff.